doc/wikis/cereal.md

```diff
@@ -0,0 +1,204 @@
+## SHOGUN `Cereal` serialization framework
+
+#### Table of Contents
+
+- [Motivation](#motivation)
+- [For SHOGUN developers](#For SHOGUN developers)
```

Comment:
I think it should be `[For SHOGUN developers](#for-shogun-developers)` to make the link work.